### ISSUE TITLE
PackageManagement.UI.Test fail on computers with ',' as decimal separator 

### ISSUE THREAD

**Issue by forderud** [NuGet/Home#1998]
NuGet does not seem to deal gracefully with differences in locale settings. The PackageManagement.UI.Test fail on my computer with Norwegian locale (',' used as decimal separator).

Setup:
- Windows 10 x64.
- Building NuGet.Client "dev" branch from GIT commit 960426fa423e9ba09d24758d109ae21bc9d2c168 (latest as of 2016-01-16).

Output when running build.ps1: 

```
  Running tests for C:\Views\NuGet\NuGet.Client\test\NuGet.Clients.Tests\PackageManagement.UI.Test\bin\Debug\PackageManagement.UI.Test.dll
  xUnit.net MSBuild Runner (32-bit .NET 4.0.30319.42000)
    Discovering: PackageManagement.UI.Test (app domain = on [shadow copy], method display = ClassAndMethod)
    Discovered:  PackageManagement.UI.Test (running 1 test case)
    Starting:    PackageManagement.UI.Test (parallel test collections = on, max threads = 8)
C:\Views\NuGet\NuGet.Client\test\NuGet.Clients.Tests\PackageManagement.UI.Test\ConverterTests.cs(36): error : PackageManagement.UI.Test.ConverterTests.DownloadCountToStringTest(num: 1200, expected: "1.2K") [FAIL] [C:\Views\NuGet\NuGet.Client\test\NuGet.Clients.Tests\PackageManagement.UI.Test\PackageManagement.UI.Test.csproj]
        Assert.Equal() Failure
                    (pos 1)
        Expected: 1.2K
        Actual:   1,2K
                    (pos 1)
          ConverterTests.cs(36,0): at PackageManagement.UI.Test.ConverterTests.DownloadCountToStringTest(Int32 num, String expected)
C:\Views\NuGet\NuGet.Client\test\NuGet.Clients.Tests\PackageManagement.UI.Test\ConverterTests.cs(36): error : PackageManagement.UI.Test.ConverterTests.DownloadCountToStringTest(num: 1230, expected: "1.23K") [FAIL] [C:\Views\NuGet\NuGet.Client\test\NuGet.Clients.Tests\PackageManagement.UI.Test\PackageManagement.UI.Test.csproj]
        Assert.Equal() Failure
                    (pos 1)
        Expected: 1.23K
        Actual:   1,23K
                    (pos 1)
          ConverterTests.cs(36,0): at PackageManagement.UI.Test.ConverterTests.DownloadCountToStringTest(Int32 num, String expected)
C:\Views\NuGet\NuGet.Client\test\NuGet.Clients.Tests\PackageManagement.UI.Test\ConverterTests.cs(36): error : PackageManagement.UI.Test.ConverterTests.DownloadCountToStringTest(num: 1234, expected: "1.23K") [FAIL] [C:\Views\NuGet\NuGet.Client\test\NuGet.Clients.Tests\PackageManagement.UI.Test\PackageManagement.UI.Test.csproj]
        Assert.Equal() Failure
                    (pos 1)
        Expected: 1.23K
        Actual:   1,23K
                    (pos 1)
          ConverterTests.cs(36,0): at PackageManagement.UI.Test.ConverterTests.DownloadCountToStringTest(Int32 num, String expected)
C:\Views\NuGet\NuGet.Client\test\NuGet.Clients.Tests\PackageManagement.UI.Test\ConverterTests.cs(36): error : PackageManagement.UI.Test.ConverterTests.DownloadCountToStringTest(num: 1239, expected: "1.24K") [FAIL] [C:\Views\NuGet\NuGet.Client\test\NuGet.Clients.Tests\PackageManagement.UI.Test\PackageManagement.UI.Test.csproj]
        Assert.Equal() Failure
                    (pos 1)
        Expected: 1.24K
        Actual:   1,24K
                    (pos 1)
          ConverterTests.cs(36,0): at PackageManagement.UI.Test.ConverterTests.DownloadCountToStringTest(Int32 num, String expected)
C:\Views\NuGet\NuGet.Client\test\NuGet.Clients.Tests\PackageManagement.UI.Test\ConverterTests.cs(36): error : PackageManagement.UI.Test.ConverterTests.DownloadCountToStringTest(num: 1234000, expected: "1.23M") [FAIL] [C:\Views\NuGet\NuGet.Client\test\NuGet.Clients.Tests\PackageManagement.UI.Test\PackageManagement.UI.Test.csproj]
        Assert.Equal() Failure
                    (pos 1)
        Expected: 1.23M
        Actual:   1,23M
                    (pos 1)
          ConverterTests.cs(36,0): at PackageManagement.UI.Test.ConverterTests.DownloadCountToStringTest(Int32 num, String expected)
C:\Views\NuGet\NuGet.Client\test\NuGet.Clients.Tests\PackageManagement.UI.Test\ConverterTests.cs(36): error : PackageManagement.UI.Test.ConverterTests.DownloadCountToStringTest(num: 12340000, expected: "12.3M") [FAIL] [C:\Views\NuGet\NuGet.Client\test\NuGet.Clients.Tests\PackageManagement.UI.Test\PackageManagement.UI.Test.csproj]
        Assert.Equal() Failure
                     (pos 2)
        Expected: 12.3M
        Actual:   12,3M
                     (pos 2)
          ConverterTests.cs(36,0): at PackageManagement.UI.Test.ConverterTests.DownloadCountToStringTest(Int32 num, String expected)
C:\Views\NuGet\NuGet.Client\test\NuGet.Clients.Tests\PackageManagement.UI.Test\ConverterTests.cs(36): error : PackageManagement.UI.Test.ConverterTests.DownloadCountToStringTest(num: 1234000000, expected: "1.23G") [FAIL] [C:\Views\NuGet\NuGet.Client\test\NuGet.Clients.Tests\PackageManagement.UI.Test\PackageManagement.UI.Test.csproj]
        Assert.Equal() Failure
                    (pos 1)
        Expected: 1.23G
        Actual:   1,23G
                    (pos 1)
          ConverterTests.cs(36,0): at PackageManagement.UI.Test.ConverterTests.DownloadCountToStringTest(Int32 num, String expected)
    Finished:    PackageManagement.UI.Test
  === TEST EXECUTION SUMMARY ===
     PackageManagement.UI.Test  Total: 11, Errors: 0, Failed: 7, Skipped: 0, Time: 0,171s
```

**Comment by emgarten** [comment]
NuGet/NuGet.Client@43e97af05d2a77aa018425ab26a8b86a2b1e90d4

**Comment by forderud** [comment]
Fixed by PR #243.
